sources/packages/web/src/views/institution/OfferingsUpload.vue

```diff
@@ -174,7 +174,17 @@
                       v-for="warning in slotProps.data.warnings"
                       :key="warning"
                     >
-                      {{ warning.warningMessage }}
+                      {{ warning.message }}
+                    </li>
+                  </ul>
+                </div>
+                <div
+                  class="alert alert-info"
```

Comment:
mm.. if its the final design we can also use `v-alert`, https://next.vuetifyjs.com/en/components/alerts/
![image](https://user-images.githubusercontent.com/77353155/201234345-07ddf63b-6a10-4dc0-84e7-773ed789cae8.png)

Reply:
Not sure how final it is but it seems worth taking a look 😉

Reply:
The change requires more style changes, I will hold changes on this for now since it is not the PR focus.